proto/iotics/api/common.proto

```diff
@@ -109,6 +109,10 @@ enum Scope {
   LOCAL = 1;
 }
 
+message RequestInfo {
```

Comment:
We could consider running an additional linter (only buf lint is run in the CI). In the past, we were using https://github.com/uber/prototool which is now archived. It was enforcing the need for comments for every message.